src/seedu/addressbook/AddressBook.java

```diff
@@ -508,7 +508,7 @@ private static String executeDeletePerson(String commandArgs) {
             return MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
         }
         final String[] targetInModel = getPersonByLastVisibleIndex(targetVisibleIndex);
-        return deletePersonFromAddressBook(targetInModel) ? getMessageForSuccessfulDelete(targetInModel) // success
+        return hasDeletedPersonFromAddressBook(targetInModel) ? getMessageForSuccessfulDelete(targetInModel) // success
                                                           : MESSAGE_PERSON_NOT_IN_ADDRESSBOOK; // not found
```

Comment:
The indentation for the ternary operator can be improved further.
Do follow the coding standards to improve this indentation.